generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -2260,6 +2260,22 @@ def load_task(self, session_name):
             metadata=metadata)
         return response.task, response.new_session_initialized
 
+    def perform_bridge_offset_nulling_cal_ex(
+            self, task, channel, skip_unsupported_channels):
+        response = self._invoke(
+            self._client.PerformBridgeOffsetNullingCalEx,
+            grpc_types.PerformBridgeOffsetNullingCalExRequest(
+                task=task, channel=channel,
+                skip_unsupported_channels=skip_unsupported_channels))
+
+    def perform_thrmcpl_lead_offset_nulling_cal(
+            self, task, channel, skip_unsupported_channels):
+        response = self._invoke(
+            self._client.PerformThrmcplLeadOffsetNullingCal,
+            grpc_types.PerformThrmcplLeadOffsetNullingCalRequest(
+                task=task, channel=channel,
+                skip_unsupported_channels=skip_unsupported_channels))
```

Comment:
As mentioned in the scrapigen PR (I think?) these methods don't exist. Let's get them added to grpc-device, and then regenerate the grpc code here. We can move that to a separate work item and PR later, if that's easier.

Reply:
> We can move that to a separate work item and PR later, if that's easier.

You can temporarily mark the new tests with `@pytest.mark.grpc_xfail(reason="...")` where ... is a brief explanation of why the test is expected to fail. (XFAIL = eXpect FAIL)

Cc: @charitylxy

Reply:
Thanks for pointing this out, I have marked it with xfail.

Reply:
I will address the change to grpc-device and regenerate the grpc code in another PR. Thanks